src/neo/Network/P2P/Payloads/ExtensiblePayload.cs

```diff
@@ -0,0 +1,87 @@
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Persistence;
+using Neo.SmartContract;
+using System;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class ExtensiblePayload : IVerifiable
```

Comment:
It should implements `IInventory` if we want to relay the message in p2p network.

Reply:
Done